apps/web/src/app/billing/services/billing-notification.service.spec.ts

```diff
@@ -40,7 +51,7 @@ describe("BillingNotificationService", () => {
   });
 
   describe("showSuccess", () => {
-    it("should show success toast", () => {
+    it("should show success toast with default title", () => {
```

Comment:
suggestion(non-blocking): add state under test (see https://contributing.bitwarden.com/contributing/testing/unit/naming-conventions)
```suggestion
    it("shows success toast with default title when provided title is empty", () => {
```